k-distribution/tests/regression-new/checkWarns/missingKlabel.k

```diff
@@ -7,7 +7,7 @@ module MISSINGKLABEL
 
   syntax MyMap ::= MyMap MyMap
       [ left, function, hook(MAP.concat), klabel(_MyMap_), symbol, assoc, comm
-      , unit(.MyMap), element(_M|->_), index(0), format(%1%n%2), unused
```

Comment:
`element` is actually already in the whitelist. 

This test was failing because the error checking iterates over the attribute HashMap without sorting. My changes then ended up causing the missing `_M|->_`  label to be reported before the missing `.MyMap` label.

I forgot to actually fix this - I'll submit a separate PR quickly to stabilize those error messages.